### PR TITLE
Modify the methods in the Space module if Citus is enabled

### PR DESCRIPTION
**Pull request by naisila**
I have added `if else` statement in the `database_size`, `relation_sizes` and `table_sizes` functions of the `Space` module in order to get the correct sizes of the database, distributed tables and distributed indexes if Citus is enabled.

### CODE REVIEW

File: lib/pghero/methods/space.rb
Comment (byucesoy) [inline lib/pghero/methods/space.rb]: Could you please format this similar to other queries below?

File: lib/pghero/methods/space.rb
Comment (byucesoy) [inline lib/pghero/methods/space.rb]: I didn't yet fully examined this query but always use EXISTS instead of COUNT(*) if you are only looking for existence.

File: lib/pghero/methods/space.rb
Comment (byucesoy) [inline lib/pghero/methods/space.rb]: * Use EXISTS
* If CASE statement becomes too long like this one, split into multiple lines for better readability.

**Review by byucesoy — CHANGES_REQUESTED** [review]
SQL queries look pretty solid. I'll have a more detailed look at them later.

File: lib/pghero/methods/space.rb
Comment (byucesoy) [inline lib/pghero/methods/space.rb]: What about
```
pg_table_size(replace('%s','$$ || tablename::regclass::text || $$', '$$ || idxname::regclass::text || $$')) 
```

instead?

File: lib/pghero/methods/space.rb
Comment (byucesoy) [inline lib/pghero/methods/space.rb]: Use lowercase here and other instances of `RESULT`. Same for `SCHEMA`

File: lib/pghero/methods/space.rb
Comment (naisila) [reply lib/pghero/methods/space.rb]: OK.

File: lib/pghero/methods/space.rb
Comment (naisila) [reply lib/pghero/methods/space.rb]: Ah yes, I forgot to change this.

File: lib/pghero/methods/space.rb
Comment (naisila) [reply lib/pghero/methods/space.rb]: OK.

File: lib/pghero/methods/space.rb
Comment (naisila) [reply lib/pghero/methods/space.rb]: 👍 👍

File: lib/pghero/methods/space.rb
Comment (naisila) [reply lib/pghero/methods/space.rb]: Nicer 👍

**Comment by naisila** [comment]
@byucesoy I have implemented the changes.

File: lib/pghero/methods/space.rb
Comment (byucesoy) [inline lib/pghero/methods/space.rb]: Minor but, you can use * instead of logicalrelid, so that this will be even shorter. (Same for other EXISTS queries)

**Review by byucesoy — CHANGES_REQUESTED** [review]
This looks good. Now only this needs tests

File: lib/pghero/methods/space.rb
Comment (naisila) [reply lib/pghero/methods/space.rb]: OK.

**Comment by naisila** [comment]
@byucesoy I have added the tests.

File: lib/pghero/methods/space.rb
Comment (byucesoy) [inline lib/pghero/methods/space.rb:68]: `sum`s should be uppercase. As a rule of thumb most SQL keywords(except type, extension, UDF names) are uppercase, remaining things are lowercase.

**Review by byucesoy — APPROVED** [review]
This looks good. We can merge this after we merge #1

**Pull request merged**
